Assets/VRTK/Scripts/Locomotion/VRTK_PlayerClimb.cs

```diff
@@ -70,6 +70,11 @@ public class VRTK_PlayerClimb : MonoBehaviour
         protected bool isClimbing;
         protected bool useGrabbedObjectRotation;
 
+        public bool IsClimbing()
```

Comment:
public method needs to be `virtual` to allow class extensions, also needs documentation.